### DISCUSSION TITLE
GHSA-45x7-px36-x8w8 go  vulnerability

### DISCUSSION THREAD

**Discussion by kavyavvv** [goharbor/harbor#19981] [General]
GHSA-45x7-px36-x8w8 go  venerability is reported for harbor core , job service and registry , when the fix for this will be avaialble ?

**Comment by stonezdj** [comment]
The [commit](https://github.com/goharbor/harbor/commit/80b3ea5501561d7be134456c9355b334877c3aaa) is already merged in main branch of harbor core, job service, as for registry, it is main branch already fixed this issue.

The CVE score is 5.9.  Maybe it will be fixed in the Harbor v2.11.0

The CVE has no impact to Harbor.

**Comment by kavyavvv** [comment]
what is the release date of 2.11?
fix for go vulnerabilities available in 2.10.2?
